agent/engine/engine_unix_integ_test.go

```diff
@@ -49,7 +49,6 @@ import (
 	"github.com/aws/amazon-ecs-agent/agent/utils/ttime"
```

Comment:
unrelated to this change, changing these to 'require' from 'assert' because failures early in a test can cause the test to hang later in the test, or in the defer statement, which makes it harder to debug. Changing to require makes the test fail early.